api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3293,6 +3293,89 @@ public void putObject(String bucketName, String objectName, InputStream stream,
   }
 
 
+  /**
+   * Uploads data from given stream as object to given bucket where the stream size is unknown.
```

Comment:
Description is wrong to this method.

Reply:
description modified :+1:

Reply:
Is this description "Uploads data from given stream as object to given bucket where the stream size is unknown." correct for this version of put object?

Reply:
No .. changed the `unknown` to `known` :+1: